connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -271,17 +256,34 @@ private static Connector instantiateConnector(Class<? extends Connector> connCla
         return result;
     }
 
-    public void stopConnector(String connName) {
+    public void stopConnectors() {
+        stopConnectors(new HashSet<>(connectors.keySet()));
+    }
+
+    public Collection<String> stopConnectors(Collection<String> connectors) {
+        final List<String> stopped = new ArrayList<>(connectors.size());
+        for (String connector: connectors) {
+            if (stopConnector(connector)) {
+                stopped.add(connector);
+            }
+        }
+        return stopped;
```

Comment:
It seems we never use the return value here? I also wonder if we should just remove the `connector` argument. This is only invoked from 2 places and I don't see a good reason to have used the list of connectors provided in the rebalance callback, which is the only place we don't use the `stopConnectors()` variant. We might want to validate the set in the callback and complain in a log if something looks wrong, but we always want to shutdown all connectors that are actively running. It would also mean we wouldn't need to check whether the connector exists in `stopConnector(String connName)`.